### ISSUE TITLE
Introduce echo control

### ISSUE THREAD

**Issue by voodooattack** [replit/jq-console#73]
Currently, the library always echoes key strokes on the prompt. 

I was able to prevent this using the following code, but it would be great if it was optional.

`console.$prompt_left = $('<div></div>');`

**Comment by amasad** [comment]
Care to mention your use case? Is it for a password prompt?

**Comment by voodooattack** [comment]
Yes. I'm building a "hacking" game, and a visible password looks terrible.

**Comment by amasad** [comment]
here is where the echo happens. https://github.com/replit/jq-console/blob/4923dbe841f5c894d133afb6e3681fa149383b2f/src/jqconsole.coffee#L793

Maybe add an option somewhere to disable it (echo: true/false). Patches welcome
